addons/interaction/functions/fnc_canOpenBackpack.sqf

```diff
@@ -0,0 +1,27 @@
+#include "..\script_component.hpp"
+/*
+ * Author: DartRuffian
+ * Checks if the player can open a unit's backpack.
+ *
+ * Arguments:
+ * 0: Target <OBJECT>
+ * 1: Player <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [cursorObject, player] call ace_interaction_fnc_canOpenBackpack
+ *
+ * Public: No
+ */
+
+params ["_player", "_target"];
+
+private _backpackContainer = backpackContainer _target;
+
+!isNull _backpackContainer &&
+{!lockedInventory _backpackContainer} &&
+{maxLoad _backpackContainer > 0} &&
+{getNumber (configOf _backpackContainer >> "disableInventory") != 1} &&
+{!(_target isKindOf QEGVAR(dragging,clone))}
```

Comment:
IMO adding it hardcoded is suboptimal, but necessary atm. I'll look into locking the inventory of clones, so it's fixed in general and we don't need to check for it explicitly here.

Reply:
Sounds good, leave it as is for now then?

Reply:
Yep.

Reply:
I think they might be always `claimed` when being dragged, so this might not be needed?

Reply:
Yea, they are claimed, so the clone can't be interacted with.